### PR TITLE
UI tweaks and ongoing integration with OsmAnd

### PR DESCRIPTION
**Pull request by MadWasp79**

### CODE REVIEW

File: app/src/main/java/net/osmand/aidl/IOsmAndAidlInterface.aidl [hunk 1]
```diff
@@ -160,4 +160,12 @@ interface IOsmAndAidlInterface {
     boolean getActiveSqliteDbFiles(out List<ASqliteDbFile> files);
     boolean showSqliteDbFile(String fileName);
     boolean hideSqliteDbFile(String fileName);
+
+    boolean setNavDrawerLogoWithIntent(in String imageUri, String packageName, String intent);
```
Comment (crimean) [inline app/src/main/java/net/osmand/aidl/IOsmAndAidlInterface.aidl]: setNavDrawerLogoWithIntent > setNavDrawerLogoWithParams
in String imageUri, String packageName, String intent > NavDrawerLogoParams

File: app/src/main/java/net/osmand/aidl/IOsmAndAidlInterface.aidl [hunk 2]
```diff
@@ -160,4 +160,12 @@ interface IOsmAndAidlInterface {
...
+    boolean setNavDrawerFooterAction(in String packageName, String intent, String appName);
```
Comment (crimean) [inline app/src/main/java/net/osmand/aidl/IOsmAndAidlInterface.aidl]: setNavDrawerFooterAction > setNavDrawerFooterWithParams
in String packageName, String intent, String appName > NavDrawerFooterParams

File: app/src/main/java/net/osmand/aidl/IOsmAndAidlInterface.aidl [hunk 3]
```diff
@@ -160,4 +160,12 @@ interface IOsmAndAidlInterface {
...
+    boolean setPointMenuEnabledIds(in List<String> ids);
+    boolean setPointMenuDisabledIds(in List<String> ids);
+    boolean setPointMenuEnabledPatterns(in List<String> patterns);
+    boolean setPointMenuDisabledPatterns(in List<String> patterns);
```
Comment (crimean) [inline app/src/main/java/net/osmand/aidl/IOsmAndAidlInterface.aidl]: if it related to context menu on the map - setMapContextMenuEnabledId, etc.

File: app/src/main/java/com/strikelines/app/OsmandCustomizationConstants.kt [hunk 4]
```diff
@@ -21,6 +21,7 @@ object OsmandCustomizationConstants {
 	const val DRAWER_HELP_ID = DRAWER_ITEM_ID_SCHEME + "help"
 	const val DRAWER_BUILDS_ID = DRAWER_ITEM_ID_SCHEME + "builds"
 	const val DRAWER_DIVIDER_ID = DRAWER_ITEM_ID_SCHEME + "divider"
+	const val DRAWER_TEST_ID = DRAWER_ITEM_ID_SCHEME + "Test 1"
```
Comment (crimean) [inline app/src/main/java/com/strikelines/app/OsmandCustomizationConstants.kt]: unneeded?

File: app/src/main/java/net/osmand/aidl/IOsmAndAidlInterface.aidl (hunk 2 above)
Comment (MadWasp79) [reply app/src/main/java/net/osmand/aidl/IOsmAndAidlInterface.aidl]: fixed

File: app/src/main/java/net/osmand/aidl/IOsmAndAidlInterface.aidl (hunk 3 above)
Comment (MadWasp79) [reply app/src/main/java/net/osmand/aidl/IOsmAndAidlInterface.aidl]: fixed

File: app/src/main/java/com/strikelines/app/OsmandCustomizationConstants.kt (hunk 4 above)
Comment (MadWasp79) [reply app/src/main/java/com/strikelines/app/OsmandCustomizationConstants.kt]: fixed

**Pull request merged**